### PR TITLE
docs: fixing broken links in the documentation

### PR DESCRIPTION
**Pull request by yerabolu**
Fixing broken links in the documentation

Signed-off-by: Spoorthy Priya Yerabolu <spoorthy.priya.yerabolu@intel.com>

### CODE REVIEW

File: docs/reference/getting_started.html [hunk 1]
```diff
@@ -350,7 +350,7 @@ <h3>3) Apply back-ported fixes<a class="headerlink" href="#apply-back-ported-fix
 <ol class="arabic simple" start="2">
 <li><p>Apply zephyr patches on to the kernel branch</p></li>
 </ol>
-<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="nb">cd</span> ../ecfwwork/zephyr_fork
+<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="nb">cd</span> ../ecfwwork/ecfwwork/zephyr_fork
```
Comment (rgundi) [inline docs/reference/getting_started.html:353]: The old link is correct. There is no need for additional "ecfwwork" folder

**Review by rgundi — COMMENTED** [review]
I see you have fixed the broken links in 2 places. The rest are all whitespace removals, right?

**Review by albertofloyd — CHANGES_REQUESTED** [review]
Need to update .rst files instead

**Comment by yerabolu** [comment]
Closing this PR as the changes need to be done on the .RST files.

**Pull request closed without merging**